parsing/parser.mly

```diff
@@ -2409,7 +2408,7 @@ constructor_declarations:
   | bar_constructor_declaration                          { [$1] }
   | constructor_declarations bar_constructor_declaration { $2 :: $1 }
 ;
-constructor_declaration:
+%inline constructor_declaration:
```

Comment:
I *wouldn't* rename here, given the separator-placement subtleties. (this `constructor_declarations` definition is asking for a `List.rev` wrapper)